docs/section11/Rule11-15.md

```diff
@@ -0,0 +1,48 @@
+# ServiceWaterHeating - Rule 11-15
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-15  
+
+**Rule Description:** "Service water loads and use shall be the same for both the proposed design and baseline building design and shall be documented by the calculation procedures described in Section 7.4.1.
+Exceptions: 
+(1) ECMs are used to reduce SWH use.. 
+(2) SWH energy consumption can be demonstrated to be reduced by reducing the required temperature of service mixed water, by increasing the temperature, or by increasing the temperature of the entering makeup water. "  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (g)  
+
+**Evaluation Context:** Each SWH use  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_component_by_id()
+
+**Applicability Checks:**
+- Each SHW use is applicable
+- look at each space in the proposed building: `for space in B_RMD...spaces:`
+  - look at each SWH use in the space, continue to rule logic: `for swh_use_p in space.service_water_heating_uses: CONTINUE TO RULE LOGIC`
+
+    ## Rule Logic:
+    - get the baseline equivalent swh use: `swh_use_b = get_component_by_id(swh_use_p.id, B_RMD)`
+    - create a value to track the rule status: `rule_status = "PASS"`
+    - create a variable to track any notes: `rule_note = """`
+    - check that swh_use_b exists, if it does, continue with rule logic: `if swh_use_b != null:`
+      - check that the use matches by checking that the use, use_units, use_multiplier_schedule, and temperature_at_fixture all match (ARE THERE OTHERS THAT NEED TO MATCH TOO?)
```

Comment:
the temperature of the distributed water shoudl also match

Reply:
Christina, are you thinking we need to trace up to the parent `ServiceWaterHeatingDistributionSystem.design_supply_temperature`?

Reply:
Yeah